hybridCarboniteMetadataService/src/main/scala/cromwell/services/metadata/hybridcarbonite/HybridMetadataServiceActor.scala

```diff
@@ -20,15 +20,16 @@ class HybridMetadataServiceActor(serviceConfig: Config, globalConfig: Config, se
   implicit val ec: ExecutionContext = context.dispatcher
 
   val carboniteMetadataServiceActorConfig: HybridCarboniteConfig = {
-    if (serviceConfig.hasPath(CarboniteConfigPath))
-      HybridCarboniteConfig.make(serviceConfig.getConfig(CarboniteConfigPath))(context.system).contextualizeErrors("read config")
-    else {
-      s"No ${getClass.getSimpleName} subservice config found for $CarboniteConfigPath".invalidNelCheck
+    val hybridCarboniteConfig = if (serviceConfig.hasPath(CarboniteConfigPath)) {
+      HybridCarboniteConfig.parseConfig(serviceConfig.getConfig(CarboniteConfigPath))(context.system).contextualizeErrors("parse config")
+    } else {
+      s"No ${getClass.getSimpleName} subservice config found for $CarboniteConfigPath.".invalidNelCheck
+    }
+
+    hybridCarboniteConfig match {
+      case Right(config) => config
+      case Left(e) => throw AggregatedMessageException("Failed to initialize HybridMetadataServiceActor", e.toList)
     }
```

Comment:
preexisting issue but it's strange this is split into two parts where the second part does almost nothing